llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -560,6 +560,8 @@ bool MemProfiler::instrumentFunction(Function &F) {
     return false;
   if (F.getName().starts_with("__memprof_"))
     return false;
+  if (F.hasFnAttribute(Attribute::DisableSanitizerInstrumentation))
```

Comment:
Also not a sanitizer, but it looks like there's not existing attribute to disable it?

Reply:
there isn't as I see, let's make `no_profile_instrument_function` be the one.